### PR TITLE
Squeezenet model 번역 추가

### PR DESCRIPTION
**Pull request by zhoonit**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [ ] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #(숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

squeezenet 모델 번역을 추가하였습니다.

### CODE REVIEW

**Comment by cepiloth** [comment]
고생하셨습니다. 
https://github.com/PyTorchKorea/tutorials-kr/blob/master/TRANSLATION_GUIDE.md 용어 사용 규칙에 softmax 도 추가 해서 용어 사용 규칙이 업데이트 되면 좋을 거 같아요.

File: pytorch_vision_squeezenet.md [hunk 1]
```diff
@@ -52,51 +52,51 @@ preprocess = transforms.Compose([
...
-Model `squeezenet1_1` is from the [official squeezenet repo](https://github.com/DeepScale/SqueezeNet/tree/master/SqueezeNet_v1.1).
-It has 2.4x less computation and slightly fewer parameters than `squeezenet1_0`, without sacrificing accuracy.
+`squeezenet1_1` 모델은 [official squeezenet repo](https://github.com/DeepScale/SqueezeNet/tree/master/SqueezeNet_v1.1) 에서 왔습니다.
+`squeezenet1_0` 수준의 정확도를 유지하며 2.4배 계산이 덜 필요하고, `squeezenet1_0`보다 변수(parameters)의 수가 적습니다.
 
-Their 1-crop error rates on imagenet dataset with pretrained models are listed below.
+Imagenet 기준으로 훈련된 모델들의 1-crop 에러율은 아래와 같습니다.
 
-| Model structure | Top-1 error | Top-5 error |
+| 모델 | Top-1 에러 | Top-5 에러 |
```
Comment (cepiloth) [inline pytorch_vision_squeezenet.md:95]: 깔끔하게 잘 하신거 같아요.
수고하셨습니다.

**Review by cepiloth — CHANGES_REQUESTED** [review]

**Review by cherimlee — COMMENTED** [review]
너무 매끄러워서 미끄러질 것 같습니다. 백점입니다.
